search.py

```diff
@@ -0,0 +1,93 @@
+"""Functions for searching"""
+import sys
+from datetime import datetime, timedelta
+
+import requests
+from requests.exceptions import ConnectionError
+
+
+def get_url(code_from, code_to, dep_date, ret_date, trip_type='TT'):
```

Comment:
this is not correct naming and docstring, this method issues actual request, not just gets/calculates URL. Consider name it as `get_website_response` or similar.